packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -269,6 +269,10 @@ const Conversation = SparkPlugin.extend({
           }, omit(options, 'id', 'user', 'url'))
         };
 
+        if ('participantAckFilter' in options) {
+          delete params.qs.includeParticipants;
```

Comment:
typically I delete the things I check for in the `if` block. What's the logic here? A code comment would be great.

Reply:
Good point on the comment.  Will add it in the next commit.

The existing code always sets the the query param `includeParticipants=false`, which negates sending the participantAckFilter.  So if the caller explicitly sets the participantAckFilter we don't want that default setting.

Reply:
@andrewholsted  , comment added in latest commit.